argo-cd-apps/overlays/staging-downstream/delete-applications.yaml

```diff
@@ -29,6 +29,7 @@ kind: ApplicationSet
 metadata:
   name: quality-dashboard
 $patch: delete
+---
```

Comment:
Kustomize build is failing for this overlay because it cannot delete `tracing-workload-otel-collector`. The reason is staging-downstream overlay is base on konflux-public-staging and applicationSet is deleted already in that overlay so trying to delete here as well end up with not found error.

Remove the deletion of `tracing-workload-otel-collector` from this overlay (line 32 to 38)

Reply:
Fixed.